department-of-veterans-affairs/caseflow
# Justin/APPEALS-56224

Resolves [Tasks related to an inactive appeal: CAVC Correspondence](https://jira.devops.va.gov/browse/APPEALS-56224)

# Description
Please explain the changes you made here.

## Acceptance Criteria

## Testing Plan

1. These first 14 include setup steps, and the testing for this specific ticket starts on step 15.
2. Log into Caseflow as an inbound ops team user.
3. Navigate to queue/correspondence. (Note: Admins and Superusers will have access to all the correspondence already, but regular iops team users may have to have a correspondence assigned to them before they will see them).
4. Select a veteran from the unassigned tab to go to the review package page
5. If necessary, add a correspondence type, and click save.
6. Scroll to the bottom and select "Create record"
7. Navigate to step 2
8. Find the "Tasks related to an existing Appeal" section and click the "yes" radio button
9. Note that the first six appeals should be inactive appeals. The first three will be assigned to "Mail" (cancelled), and the next three will be assigned to "Post-decision" (closed).
10. Choose one of these appeals and verify that it is inactive. You can verify this by following the next three steps.
11. Scroll down and click the "+ Add tasks" button
12. Click the task dropdown.
13. Verify that there are only 12 options in this list. A quick way to check is to make sure the list does not contain "Change Of Address", as this is only available on active appeals
14. On this inactive appeal, we want to verify the CAVC Correspondence task.
15. Verify the CAVC option exists in the list of available tasks for inactive appeals, as this option should be available for any appeal.
16. Choose the cavc task and add notes into the notes section.
17. Continue to step 3 of the intake form, and confirm that the CAVC Correspondence tasks is visible in the Tasks Related to and Existing Appeal section. (Note, you may want to grab the address from the address bar at this point).
18. Submit the correspondence, and confirm the submission. The record should successfully be created.
19. Next, paste the previously-copied URL into the address bar, removing the /intake portion, and go to that page
20. The linked appeal should appear in the "Existing Appeals" section, and it should be assigned to CAVC Litigation Support
